pkg/payload/task_graph_test.go

```diff
@@ -949,7 +949,6 @@ func TestRunGraph(t *testing.T) {
 			callbacks: map[string]callbackFn{
 				"a": func(t *testing.T, name string, ctx context.Context, cancelFn func()) error {
 					cancelFn()
-					time.Sleep(time.Second)
```

Comment:
I'd added this with the test-case back in eaa3d190d93d66 (#255).  I didn't talk about the `Sleep` specifically in the commit message, but my guess is that what I was aiming for was "make sure `b` isn't running because the task-graph runner knows `a` failed" to avoid "`b` would have run, except the runner got lucky and raced closed before it got around to completing".  It's been in place for a long time now though, and I'm fine dropping the "be-extra certain" sleep at this point in order to allow us to run tests more quickly.